### PR TITLE
Add test cases to verify eu-ES(Basque-Spain)

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Added test cases to verify eu-ES (Basque-Spain)
* timezone: Europe/Madrid
* Added test case feature: LocaleInfo,Script, String(plural) LocaleMatch,  DateFmt, DurFmt, DateRngFmt, List, Unit, firstDayofWeek, clock, calendar,Month/Weekday Translation, NumFmt,

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/test/date/testcalendar.js [hunk 1]
```diff
@@ -23,14 +23,14 @@ if (typeof(LocaleInfo) === "undefined") {
 
 module.exports.testcalendar = {
     testDefaultCalendar_GregorianCaseAll: function(test) {
-        test.expect(193);
+        test.expect(195);
         var localeList = ["ar-AE","ar-BH","ar-DJ","ar-DZ","ar-EG","ar-IQ","ar-JO","ar-KW","ar-LB","ar-LY","ar-MA","ar-MR",
-        "ar-OM","ar-QA","ar-SA","ar-SD","ar-SY","ar-TN","ar-YE","as-IN","az-Latn-AZ","bg-BG","bn-IN","bs-Latn-BA,bs-Latn-ME",
+        "ar-OM","ar-QA","ar-SA","ar-SD","ar-SY","ar-TN","ar-YE","as-IN","az-Latn-AZ","be-BY","bg-BG","bn-IN","bs-Latn-BA,bs-Latn-ME",
```
Comment (ehoogerbeets) [inline js/test/date/testcalendar.js:28]: Are all these locale test branches independent of each other? If so, be-BY should not appear in this list in this branch

File: js/test/date/testdatefmt_eu_ES.js [hunk 2]
```diff
@@ -0,0 +1,1788 @@
...
+            locale: "eu-ES",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "2011(e)ko ira. 29(a)");
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_eu_ES.js:81]: From perusing various basque web sites, it seems that the (e) and the (a) are optional and are used when formatting the correct gender/case of the number. Sort like when you write "word(s)" in English to indicate a possible plural. That means they are optional parts that depend on the number that is being formatted. I am not sure this date format is acceptable to Basque people, or if it looks really awkward. Please have this verified by any Basque LG members you can find.

Comment (gouniLee) [reply js/test/date/testdatefmt_eu_ES.js:81]: Regarding `eu-ES` locale, it's not officially required to support at this locale now, and it's just preparing for reference platform perspective. so It's hard to find(or contact) Basque people. The data is directly from CLDR without modifying it. I think It's better to ask to CLDR.

File: js/test/date/testcalendar.js (hunk 1 above)
Comment (gouniLee) [reply js/test/date/testcalendar.js]: Actually, I wanted to separate PRs per local to see the change clearly. but it modifies the same files at the same time and can't expect when it's merged. Probably it's not a good idea to make a PR per locale.. I need to make test cases for 8 locales. probably making one PR would be better.

File: js/test/date/testdatefmt_eu_ES.js (hunk 2 above)
Comment (ehoogerbeets) [reply js/test/date/testdatefmt_eu_ES.js:81]: Different PRs are better. It is easier to see the differences, and merging the one file is not so difficult.

Comment (gouniLee) [reply js/test/date/testdatefmt_eu_ES.js:81]: ok. I'll do that way.

**Pull request merged**